### ISSUE TITLE
Crash in MPNotification.m line 100

### ISSUE THREAD

**Issue by PanJ** [mixpanel/mixpanel-iphone#287]
The error was
```
Fatal Exception: NSInvalidArgumentException
*** -[NSURL initWithScheme:host:path:]: path 1435990860000.418153.2ea8fffb503387cd26500ddf34d67ad1@2x.jpg is not absolute.
```
Here is raw output from crashlytics
```
Thread : Fatal Exception: NSInvalidArgumentException
0  CoreFoundation                 0x00000001865c9e48 __exceptionPreprocess + 132
1  libobjc.A.dylib                0x0000000196d080e4 objc_exception_throw + 60
2  CoreFoundation                 0x00000001865c9d88 -[NSException initWithCoder:]
3  Foundation                     0x00000001874aa1b8 -[NSURL(NSURL) initWithScheme:host:path:] + 144
4  Zon                            0x000000010027c9d4 +[MPNotification notificationWithJSONObject:] (MPNotification.m:100)
5  Zon                            0x00000001002a860c __58-[Mixpanel checkForDecideResponseWithCompletion:useCache:]_block_invoke (Mixpanel.m:1241)
6  libdispatch.dylib              0x000000019734d3ac _dispatch_call_block_and_release + 24
7  libdispatch.dylib              0x000000019734d36c _dispatch_client_callout + 16
8  libdispatch.dylib              0x00000001973574c0 _dispatch_queue_drain + 1216
9  libdispatch.dylib              0x0000000197350474 _dispatch_queue_invoke + 132
10 libdispatch.dylib              0x0000000197359224 _dispatch_root_queue_drain + 664
11 libdispatch.dylib              0x000000019735a75c _dispatch_worker_thread3 + 108
12 libsystem_pthread.dylib        0x00000001975292e4 _pthread_wqthread + 816
13 libsystem_pthread.dylib        0x0000000197528fa8 start_wqthread + 4
```

**Comment by samgreen** [comment]
Hey @PanJ so sorry about this issue. I'll look in to it today and get back to you.

**Comment by knutigro** [comment]
Hi, any news on this issue? It's a quite common crash in my app for the moment.

**Comment by samgreen** [comment]
This will be resolved in the next release.

**Comment by samgreen** [comment]
This is now released as v2.9.2, thanks for your patience!
